crates/sage-api/src/requests/data.rs

```diff
@@ -51,24 +52,59 @@ pub struct GetSyncStatusResponse {
 
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 #[cfg_attr(feature = "tauri", derive(specta::Type))]
-pub struct GetXchCoins {}
+#[serde(rename_all = "snake_case")]
+pub enum CoinSortMode {
+    ParentCoinId,
```

Comment:
I don't think this is particularly useful

Reply:
renamed and bound to coin_id in the query